app-frontend/config/webpack.dev.config.js

```diff
@@ -0,0 +1,29 @@
+var webpack = require('webpack'),
+    config = require('./webpack.common.config.js'),
+    host = process.env.WEBPACK_DEV_SERVER || 'http://localhost:8080';
+
+config.devtool = 'inline-source-map';
```

Comment:
It's probably safe to go to `eval` here -- especially since build times are kind of slow already with a small application.